generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -39,7 +39,9 @@
 import <%=packageName%>.service.util.RandomUtil;
 <%_ } _%>
 import <%=packageName%>.service.dto.UserDTO;
+<%_ if (authenticationType !== 'oauth2') { _%>
```

Comment:
@mraible are the changes in this file required? the react branch is up to date with changes in master branch so I dont understand why this change shows up here?

Reply:
Me neither, I'll revert changes.